package-lock.json

```diff
@@ -6821,9 +6821,9 @@
       "dev": true
     },
     "eslint-plugin-testing-library": {
-      "version": "3.7.0",
```

Comment:
@Averethel Can you elaborate on this I only see a change in the lock file and none in the package json so this looks a bit stange to me... 
Thanks 😄

Reply:
It's a dependency of a dependency. I forced lower version since 3.7 has eslint < 7 in peer deps.
It's one of those things where I would love to have deps pinned to catch that updates in the package